### ISSUE TITLE
line 25 […] <class 'TypeError'>: argument of type 'NoneType' is not iterable

### ISSUE THREAD

**Issue by galantra** [Arthur-Milchior/anki-number-of-selected-tags#1]
Hello Arthur,

an error occured when I  (unsuccessfully) tried to open the browser.

### I checked:

1.  if the _Multiple Windows_ add-on causes the error
2. if the _Advanced Browser_ add-on causes the error
3. if a combination of both causes the error
4. interference with any other shortcut

The error doesn't go away when they are disabled (or deleted).

I want to stress that this error is low-priority for me. Nevertheless I want to inform you.

```
Error 
An error occurred. Please start Anki while holding down the shift key, which will temporarily disable the add-ons you have installed. 
If the issue only occurs when add-ons are enabled, please use the Tools>Add-ons menu item to disable some add-ons and restart Anki, repeating until you discover the add-on that is causing the problem. 
When you've discovered the add-on that is causing the problem, please report the issue on the add-ons section of our support site. 
Debug info:
Anki 2.1.15 (6661ba8a) Python 3.6.7 Qt 5.12.1 PyQt 5.11.3
Platform: Windows 10
Flags: frz=True ao=True sv=2

Caught exception:
  File "aqt\main.py", line 855, in onBrowse
  File "C:\Users\user\PortableApps\AnkiPortable\Anki2\addons21\354407385\multiple.py", line 68, in open
    return function(name,*args)
  File "C:\Users\user\PortableApps\AnkiPortable\Anki2\addons21\354407385\multiple.py", line 80, in openMany
    instance = creator(*args)
  File "C:\Users\user\PortableApps\AnkiPortable\Anki2\addons21\874215009\advancedbrowser\core.py", line 276, in __init__
    origInit(self, mw)
  File "aqt\browser.py", line 404, in __init__
  File "aqt\browser.py", line 467, in setupMenus
  File "anki\hooks.py", line 29, in runHook
  File "C:\Users\user\PortableApps\AnkiPortable\Anki2\addons21\1918380616\__init__.py", line 79, in setupMenu
    a.setShortcut(QKeySequence(getUserOption("Shortcut of 'Number of tags'", "Ctrl+t")))
  File "C:\Users\user\PortableApps\AnkiPortable\Anki2\addons21\1918380616\config.py", line 25, in getUserOption
    if key in userOption:
<class 'TypeError'>: argument of type 'NoneType' is not iterable
```

**Comment by Arthur-Milchior** [comment]
> The error doesn't go away when they are disabled (or deleted). 
When what is disabled/deleted ?
If you delete the add-on and restart anki I assume that at least the error message change. Isn't it the case ? (You need to restart anki after disabling add-on)

Did you edit the add-on config ? Did you do anything special to it ? Because it seems that it can't access the configuration file or the default configuration, and it makes no sens.

**Comment by galantra** [comment]
> When what is disabled/deleted ?

The _Multiple Windows_ add-on and the _Advanced Browser_ add-on. Yes, I restarted Anki.

> If you delete the add-on and restart anki I assume that at least the error message change. Isn't it the case ? (You need to restart anki after disabling add-on)

The error message did change when I disabled or deleted the _Multiple Windows_ and _Advanced Browser_ add-on(s) (to be exact: moved the particular add-on's folder out of the _addons21_ folder). 
I did not disable nor delete the High Yield Tags add-on.

> Did you edit the add-on config ? Did you do anything special to it ?

Not that I remember. I did, though, make the first installation on Anki 2.1.15.

Now, entered the code 1918380616 into the add-on window and thereby downloaded the add-on again.

Also, I've installed Anki 2.1.20. I entered 1918380616 into _Get Add-ons_ again (because it told me that my existing addon version requires Anki 2.1.19 or below. So I assume there are two versions of the add-on 1918380616). I restarted Anki.

Now I can access the browser window and High Yield Tags is working, too.
